internal/datasource/streaming_data_source_events.go

```diff
@@ -16,7 +16,7 @@ var (
 	deleteDataRequiredProperties = []string{"path", "version"} //nolint:gochecknoglobals
```

Comment:
This is just a reversion of recent changes that made these public (for use in datasourcev2), but are no longer necessary.